index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
+    </div>
+    <nav class="navlinks">
+      <img src="/Img/Hlogo.png">
+    </nav>
+  </header>
+  <section class="headline-main">
+    <div class="headline">
```

Comment:
Please let's add the remaining section's `background` images to this section so as to match the given template. 🛠💪

Figma 👇 
![image](https://user-images.githubusercontent.com/78251927/235113602-ca978c02-f689-4138-90e7-e7069188b814.png)

Your design 👇 
![image](https://user-images.githubusercontent.com/78251927/235113656-85a3829c-9599-486c-8a36-61d64f6d89cd.png)

Reply:
Hi @shurichmond, I realized that the background images are all displaying properly on the mobile view and the error is from my end. You can ignore this particular comment. 

Thanks 👍